wherehows-docker/docker-compose.yml

```diff
@@ -36,3 +49,4 @@ services:
     command: dockerize -wait tcp://wherehows-mysql:3306 -timeout 120s bin/playBinary
```

Comment:
You'll also need to set the various ES configs in application.conf via env vars: https://github.com/linkedin/WhereHows/blob/master/wherehows-frontend/conf/application.conf#L66